example/vllm-serve/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mistral-7b
+  namespace: default
+  labels:
+    app: mistral-7b
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mistral-7b
+  template:
+    metadata:
+      labels:
+        app: mistral-7b
+    spec:
+      volumes:
+      - name: dev-kfd
+        hostPath:
+          path: /dev/kfd
+      - name: dev-dri
+        hostPath:
+          path: /dev/dri
```

Comment:
Have you tried your example without these hostPath?  It should work without these as the device plugin does these mounting for you.

Reply:
Yes. It works after remove the hostPath and mountPath of /dev/kfd and /dev/dri. Does it means these device nodes are mount to container by default by ROCm/k8s-device-pluging?

Reply:
I believe kubelet does the mounting with information from the ROCm/k8s-device-plugin